### PR TITLE
Issue #5662 Fix text on change button

### PR DESCRIPTION
**Pull request by giniouxe**

### CODE REVIEW

**Comment by jhass** [comment]
Hey, please check [this comment](https://github.com/diaspora/diaspora/issues/5662#issuecomment-74240968) in the issue ;)

File: config/locales/diaspora/en.yml [hunk 1]
```diff
@@ -1319,6 +1319,7 @@ en:
     privacy_settings:
       title: "Privacy settings"
       strip_exif: "Strip metadata such as location, author, and camera model from uploaded images (recommended)"
+      change: "Change"
```
Comment (goobertron) [inline config/locales/diaspora/en.yml]: I think this addition can be removed as you're using an existing string.

**Comment by goobertron** [comment]
Thanks for your PR - the first of many, I hope! It's great to see a new contributor.

**Comment by svbergerem** [comment]
Hooray, a new contributor! :)

Like @goobertron said, please remove the line from `config/locales/diaspora/en.yml` you added as you don't need it anymore. Squash the commits afterwards into one and then this should be ready to merge.

**Comment by jhass** [comment]
Let's extract the single wanted commit

```bash
git remote add upstream git://github.com/diaspora/diaspora.git
git fetch upstream
git checkout 5662-privacy-button-text
git reset --hard upstream/develop
git cherry-pick a8a44e6f87f78791c1e2d3814299f34ec4337bb0
git push -f origin 5662-privacy-button-text
```

In the future you can edit a commit wit `git commit --amend` and then force push it into your feature branch (`git push -f`).

**Comment by giniouxe** [comment]
Sorry @jhass ! Just corrected this point with a new commit.
Hope it's OK now.

**Comment by giniouxe** [comment]
And I'm correcting the mess in the commits right now.

**Comment by giniouxe** [comment]
@goobertron && @svbergerem Thanks for the welcome messages :)

**Comment by jhass** [comment]
Perfect, thank you!

**Comment by goobertron** [comment]
Hooray! :-)

**Comment by Flaburgan** [comment]
@giniouxe heeey! Welcome aboard! Guys let me introduce you Aurélie who discovered diaspora* with framasphere (I guess). Aurélie don't hesitate if you have any question, you can join #diaspora-dev or #diaspora-fr on IRC Freenode.

**Comment by giniouxe** [comment]
@Flaburgan Thanks :)

**Pull request merged**